components/brave_wallet/browser/eth_abi_decoder.cc

```diff
@@ -142,11 +136,11 @@ absl::optional<std::string> GetBoolFromData(const std::vector<uint8_t>& input,
 absl::optional<std::string> GetBytesHexFromData(
     const std::vector<uint8_t>& input,
     size_t offset) {
-  auto pointer = GetSizeFromData(input, offset);
+  auto pointer = GetUintFromData<size_t>(input, offset);
```

Comment:
Strictly speaking this behaves differently for 32 and 64 bit platforms and we parse platform independent format. But this should work as we use these values as offsets and lengthts within relatively small payloads

Reply:
Good point. The behaviour doesn't change from `GetSizeFromData` though. We only parse calldata for known function selectors today, so the risk of exceeding the limits of `size_t` is very low.